src/content/docs/style-guide/images/image-annotations.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Best practices for using image annotations
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+---
+
+import styleguideExampleCalloutBox from 'images/style-guide_screenshot-full_example_callout_box.png'
+
+import apisanddataNerdGraphDashboardEntity from 'images/apis-and-data_screenshot-full_NerdGraph-dashboard-entity.png'
+
+import styleguideExampleArrow from 'images/style-guide_screenshot-full_example_arrow.png'
+
+Image annotations are text and shapes in screen captures or diagrams that help to highlight information and focus attention on the most important aspects of the screenshot. Avoid the use of text inside a screenshot, use [captions](/docs/style-guide/images/screenshots-images/#captions) instead.
+
+If you need to use image annotations:
+
+* Be concise.
+* Make sure annotations are easy to distinguish from UI text. For example, you may need to offset your boxes so they don't look like UI elements.
+* Annotations aren't translated, so use them sparingly.
+* Don't use an annotation at the top that acts like a title. Use a caption.
+* Use our [New Relic SnagIt theme](https://github.com/newrelic/newrelic-doc-tools) (link only available to New Relic employees) to keep your annotations consistent. 
+
+  To install the New Relic SnagIt theme:
+    1. Ensure you're running the latest version of SnagIt.
+    2. In GitHub Desktop, clone the [repo](https://github.com/newrelic/newrelic-doc-tools) locally.
+    3. Open SnagIt and select any tool to open the right nav.
+    4. Under the **Quick Styles** pane in the right nav, select the **Theme** gear icon, then **Import**.
+
+## Callout boxes [#callout-boxes]
+
+Use callouts to highlight an element within a screenshot. If possible, don't add text to screenshots, that's what our docs are for. Callout boxes should not obstruct any of the elements they're calling out. For example, a callout box around a UI button shouldn't cover any part of the button. If possible, a 2-pixel clearance should be given between the UI element and the callout box.
+
+This is the format for adding callout boxes:
+  * Shape: square or rectangle, transparent fill, no border-radius.
+  * Border color: #00CE7C
+  * Line width: 5px
+  * Opacity: 100%
+  * Line style: solid
+  * No shadows
+
+<img
+  title="Example of a callout box"
+  alt="Example of a callout box"
+  src={styleguideExampleCalloutBox}
+/>
+
+<figcaption>
+  This is an example of a callout box.
+</figcaption>
+
+## Callout text [#callout-text]
+
+Screenshots shouldn't require additional text on them in 99% of all situations; it's the job of the written documentation itself to explain what is being represented in the screenshot. However, if a text box is required, it shouldn't overlap any essential elements of the UI.
+
+In case of adding text, follow this format:
```

Comment:
@nbaenam maybe revise like this: "If you have to add text to a screenshot, follow this format:"

Reply:
What did you think of this idea?

Reply:
Yes, much better. Added, thanks!